doc/manual/43-backup-commands.en.md

```diff
@@ -198,6 +198,13 @@ command.
 
 The recovery command has several options that modify the command behavior.
 
+> **NOTE:**
+> If you’re not sure whether the backup can be automatically removed by
+> retention policies during recovery, avoid issuing the `recover` command
+> right away. Instead, first mark the backup with `keep` and specify the
+> `target`. Once the recovery process is complete, if you don’t want to
+> keep the backup beyond the retention policy, you can then release it.
```

Comment:
Reworded for clarity
```suggestion
> If you're unsure whether the backup you want to recover from will be
> automatically deleted by retention policies during recovery,
> mark it using the [keep command](#keep) with the appropriate target.
> Once the recovery process is complete, if you don’t want to
> keep anymore the backup beyond the retention policy, you can remove
> the `keep` annotation issuing the [keep command](#keep) again
> using the `--release` option
```
@andremagui @martinmarques @barthisrael , thoughts?

Reply:
Yes, i agree! I thought about fixing 'full' as the target in the note but i was not sure about this.

Reply:
@barthisrael @martinmarques Any comments on this?

Reply:
TBH i think this is an assumption that is valid for people that did not understand the barman recovery process. I mean, if the user does not know what it needs for the recovery process to be complete, than it should keep the backup (source backup) until the new PostgreSQL instance is running (from the destination recovered backup) and tested the PIT target chosen. Does it make sense? If so, we can add more info about this.

Reply:
that's why i wrote "recovery process" and not recovery... but looks like is confusing 😄 I don't have a better Idea atm for wording differently

Reply:
Maybe something like

"Once the recovery process is complete and your Postgres instance has reached the desired recovery target..."